tests/Keboola/Csv/CsvFileTest.php

```diff
@@ -289,4 +289,52 @@ public function testNonStringWrite()
 
 		$csvFile->writeRow($row);
 	}
+
+	public function testSkipsHeaders()
+	{
+		$fileName = __DIR__ . '/_data/simple.csv';
+
+		$csvFile = new \Keboola\Csv\CsvFile($fileName, CsvFile::DEFAULT_DELIMITER, CsvFile::DEFAULT_ENCLOSURE, CsvFile::DEFAULT_ENCLOSURE, 1);
+		$this->assertEquals([
+			['15', '0'],
+			['18', '0'],
+			['19', '0'],
+		], iterator_to_array($csvFile));
+
+	}
+
+    public function testSkipNoLines()
+    {
+        $fileName = __DIR__ . '/_data/simple.csv';
+
+        $csvFile = new \Keboola\Csv\CsvFile($fileName, CsvFile::DEFAULT_DELIMITER, CsvFile::DEFAULT_ENCLOSURE, CsvFile::DEFAULT_ENCLOSURE, 0);
+        $this->assertEquals([
+            ['id', 'isImported'],
+            ['15', '0'],
+            ['18', '0'],
+            ['19', '0'],
+        ], iterator_to_array($csvFile));
+
```

Comment:
Why are here these empty lines?

Reply:
i'll fix the CS in next PR, there are too many issues now

Reply:
yes, i noticed, but why are you introducing new cs issue?:)